### PR TITLE
[INFRA-130] slim

### PR DESCRIPTION
**Pull request by kotchaosu**
```
ruby-codebox2                     latest              466f33376bdb        23 seconds ago      438.6 MB
quay.io/syncano/ruby-codebox      latest              848b1040bfc7        6 days ago          723.8 MB
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,19 +1,31 @@
-FROM ruby:2.2
-
+FROM ubuntu:trusty
 MAINTAINER "Syncano DevOps Team" <devops@syncano.com>
 
-ENV LAST_REFRESHED 2015-12-07
+ENV LAST_REFRESHED 2015-12-17
+ENV API_ROOT https://v4.hydraengine.com
```
Comment (23doors) [inline Dockerfile:5]: That doesn't seem right btw.

Comment (kotchaosu) [inline Dockerfile:5]: Oh yes... Don't know why this wasn't changes so far.

Comment (ml) [inline Dockerfile:5]: What should it be? `https://api.syncano.io`?

Comment (kotchaosu) [inline Dockerfile:5]: @ml yes, I made it in next PR

**Pull request merged**